### PR TITLE
Readmes rach java5

### PR DESCRIPTION
**Pull request by Rachael-E**
This one seemed much more straight forward than batch no4! @sorenoid please could you take a first look?

### CODE REVIEW

File: java/sketch-editor/README.md [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
+
+## How to use the sample
+
+Choose which geometry type to sketch from one of the available buttons along the bottom of the screen. Choose from points, multipoints, polylines, polygons, freehand polylines, and freehand polygons.
+
+Use the buttons at the top of the screen to: undo or redo change made to the sketch whilst in sketch mode, and to save the sketch to the graphics overlay.
+
+## How it works
+
+1.  Create a `SketchEditor` and set it to the MapView with `mapView.setSketchEditor(sketchEditor)`.
```
Comment (sorenoid) [inline java/sketch-editor/README.md]: just a suggestion. I've seen this in a few places and find it a bit awkward. 

```suggestion
1.  Create a `SketchEditor` and pass it to the MapView with `mapView.setSketchEditor(sketchEditor)`.
```

**Comment by Rachael-E** [comment]
thanks @sorenoid, made those suggested changes. happy for this to go to second review?

File: java/sketch-editor/README.md [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
+## How to use the sample
+
+Choose which geometry type to sketch from one of the available buttons along the bottom of the screen. Choose from points, multipoints, polylines, polygons, freehand polylines, and freehand polygons.
+
+Use the buttons at the top of the screen to: undo or redo change made to the sketch whilst in sketch mode, and to save the sketch to the graphics overlay.
+
+## How it works
+
+1.  Create a `SketchEditor` and pass it to the MapView with `mapView.setSketchEditor(sketchEditor)`.
+2.  Use `SketchEditor.start(SketchCreationMode)` to start sketching. If editing an existing graphic's geometry, use `SketchEditor.start(graphic.getGeometry)`.
```
Comment (hud10837) [inline java/sketch-editor/README.md]: I think it's worthwhile to keep in, but the sample doesn't seem to have any instances of editing an existing graphic's geometry.

Comment (Rachael-E) [reply java/sketch-editor/README.md]: good shout: in the Java SE sample we can edit the geometry. I've changed the format here slightly to make it look like an optional comment.

**Comment by Rachael-E** [comment]
thanks both! merging now.

**Pull request merged**